base/src/main/java/proguard/optimize/inline/lambdalocator/Lambda.java

```diff
@@ -0,0 +1,50 @@
+package proguard.optimize.inline.lambdalocator;
+
+import proguard.classfile.Clazz;
+import proguard.classfile.Method;
+import proguard.classfile.attribute.CodeAttribute;
+import proguard.classfile.instruction.ConstantInstruction;
+
+/**
+ * A simple class containing information about a lambda.
```

Comment:
Can you clarify more in the code/comments what you mean with `lambda`. There is a difference between the structure of java and kotlin lambda's. As far as I understood you're looking at kotlin lambda's. Can you describe more what exact patterns you're representing with this class.